### ISSUE TITLE
Pipeline not created

### ISSUE THREAD

**Issue by kamalnayan10** [rafaelcaricio/vid2img#3]
Hi so I was using this crate for a personal project and it showed this error `
Cannot start pipeline: StateChangeError
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\video_to_ascii.exe` (exit code: 101)
` emerging at line 135 in video_stream.rs

**Comment by rafaelcaricio** [comment]
Could you set this environment variable `GST_DEBUG=4` and paste here the console output? This will log all information about the pipeline transitions and is essential to help you with this issue.

**Comment by kamalnayan10** [comment]
thread 'main' panicked at src\main.rs:7:63:
called `Result::unwrap()` on an `Err` value: IoError(Os { code: 2, kind: NotFound, message: "The system cannot find the file specified." })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\video_ascii.exe` (exit code: 101)

this is the error after setting the environment variable

**Comment by rafaelcaricio** [comment]
Sounds like the file could not be found. Are you providing the correct path?

**Comment by kamalnayan10** [comment]
yeah , i tried to put the relative path first , and then put in the full path as well, it still doesn't work , could it be due to the fact that im on windows?

**Comment by rafaelcaricio** [comment]
It could be, indeed.
